packages/terra-compact-interactive-list/src/subcomponents/Cell.jsx

```diff
@@ -62,6 +103,34 @@ const Cell = (props) => {
 
   const isResponsive = checkIfColumnIsResponsive(flexGrow, width);
 
+  useEffect(() => {
+    if (cellRef?.current) {
+      const interactiveChildren = getFocusableElements(cellRef?.current);
+      // if no interactive elements
+      if (interactiveChildren?.length === 0) {
+        setIsSelectableCell(true);
+      }
+    }
+  }, [cellRef]);
+
+  const handleKeyDown = (event) => {
+    // does not need setFocusedCell call as the cell focused by key is tracked in CompactInteractiveList component
+    const key = event.keyCode;
+    if (key === KeyCode.KEY_SPACE && isSelectableCell) {
```

Comment:
So a cell cannot be selected if it doesn't have interactable elements?

Reply:
Per @eawww at that moment (unless it changes in the future) the cell can NOT be selected If there is an interactable element in it (and there can be only one intreactive element per cell).